src/rcx/src/netRC.cpp

```diff
@@ -2017,7 +2015,9 @@ void extMain::makeBlockRCsegs(const char* netNames,
     }
 
   */
-  _modelTable->resetCnt(0);
+  while (_modelTable->notEmpty()) {
```

Comment:
Do you know that all instances of extRCModel are in _modelTable?

Reply:
There is an instance in `Ext::rc_estimate` but I added a delete for it.

The remaining problematic instance is allocated in `Ext::init_rcx_model`, and set to `extMain::_currentModel`. The issue is that this variable can also be set to an element of `_modelTable`. I can add tracking through a boolean variable to correctly free this instance and avoid risking a double free